packages/cozy-harvest-lib/package.json

```diff
@@ -14,10 +14,12 @@
     "url": "https://github.com/cozy/cozy-libs/issues"
   },
   "scripts": {
-    "test": "jest --verbose --coverage",
-    "build": "babel src -d dist",
+    "build": "babel src -d dist --copy-files",
+    "prebuild": "yarn tx",
     "prepublishOnly": "yarn build",
-    "watch": "yarn build --watch"
+    "test": "jest --verbose --coverage",
+    "tx": "tx pull --all || true",
+    "watch": "babel src -d dist --copy-files --watch"
```

Comment:
No lint script?

Reply:
I will add the lint script in a dedicated PR if you don't mind